ui/next/ts/app.tsx

```diff
@@ -0,0 +1,14 @@
+/// <reference path="../../typings/main.d.ts" />
```

Comment:
Are we still going to need this everywhere? Is there some way to specify it in the tsconfig.json?